<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dial_AsmProg</class>
 <widget class="QDialog" name="Dial_AsmProg">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>498</width>
    <height>498</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Asm FPGA Programming</string>
  </property>
  <widget class="QLineEdit" name="lineEdit_Fname">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>30</y>
     <width>321</width>
     <height>21</height>
    </rect>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QToolButton" name="toolButton_File">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>30</y>
     <width>26</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>...</string>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>0</y>
     <width>421</width>
     <height>71</height>
    </rect>
   </property>
   <property name="title">
    <string>File Select</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_EPCS">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>110</y>
     <width>113</width>
     <height>21</height>
    </rect>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>110</y>
     <width>41</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Type</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QCheckBox" name="checkBox_Erase">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>110</y>
     <width>86</width>
     <height>20</height>
    </rect>
   </property>
   <property name="layoutDirection">
    <enum>Qt::RightToLeft</enum>
   </property>
   <property name="text">
    <string>Erased</string>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox_2">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>70</y>
     <width>421</width>
     <height>81</height>
    </rect>
   </property>
   <property name="title">
    <string>EPCS</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_Do">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>170</y>
     <width>113</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>Do It</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_Close">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>210</y>
     <width>113</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>Close</string>
   </property>
  </widget>
  <widget class="QProgressBar" name="progressBar">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>252</y>
     <width>431</width>
     <height>31</height>
    </rect>
   </property>
   <property name="value">
    <number>0</number>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>260</y>
     <width>171</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Progress</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="textEdit_Msg">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>290</y>
     <width>431</width>
     <height>191</height>
    </rect>
   </property>
  </widget>
  <widget class="QCheckBox" name="checkBox_Asm_4">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>160</y>
     <width>86</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Asm 4</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="checkBox_Asm_5">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>180</y>
     <width>86</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Asm 5</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="checkBox_Asm_9">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>180</y>
     <width>86</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Asm 9</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="checkBox_Asm_7">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>220</y>
     <width>86</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Asm 7</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="checkBox_Clear">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>180</y>
     <width>61</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Clear</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="checkBox_Asm_6">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>200</y>
     <width>86</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Asm 6</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="checkBox_Asm_11">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>220</y>
     <width>86</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Asm 11</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="checkBox_Asm_10">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>200</y>
     <width>86</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Asm 10</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="checkBox_All">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>160</y>
     <width>61</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>All</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="checkBox_Asm_3">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>220</y>
     <width>86</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Asm 3</string>
   </property>
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QCheckBox" name="checkBox_Asm_1">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>180</y>
     <width>86</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Asm 1</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="checkBox_Asm_2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>200</y>
     <width>86</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Asm 2</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="checkBox_Asm_8">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>160</y>
     <width>86</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Asm 8</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="checkBox_Asm_0">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>160</y>
     <width>86</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Asm 0</string>
   </property>
  </widget>
  <zorder>groupBox</zorder>
  <zorder>groupBox_2</zorder>
  <zorder>lineEdit_Fname</zorder>
  <zorder>toolButton_File</zorder>
  <zorder>lineEdit_EPCS</zorder>
  <zorder>label</zorder>
  <zorder>checkBox_Erase</zorder>
  <zorder>pushButton_Do</zorder>
  <zorder>pushButton_Close</zorder>
  <zorder>progressBar</zorder>
  <zorder>label_2</zorder>
  <zorder>textEdit_Msg</zorder>
  <zorder>checkBox_Asm_4</zorder>
  <zorder>checkBox_Asm_5</zorder>
  <zorder>checkBox_Asm_9</zorder>
  <zorder>checkBox_Asm_7</zorder>
  <zorder>checkBox_Clear</zorder>
  <zorder>checkBox_Asm_6</zorder>
  <zorder>checkBox_Asm_11</zorder>
  <zorder>checkBox_Asm_10</zorder>
  <zorder>checkBox_All</zorder>
  <zorder>checkBox_Asm_3</zorder>
  <zorder>checkBox_Asm_1</zorder>
  <zorder>checkBox_Asm_2</zorder>
  <zorder>checkBox_Asm_8</zorder>
  <zorder>checkBox_Asm_0</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
